### PR TITLE
feat(dialog): stories improvements and LuModal mapping

### PR DESCRIPTION
**Pull request by Supamiu**
## Description

This PR adds two stories that will be used for RC testing documentation purpose and better manual testing tooling.

It also adds mappings to `LuModal` service to make it use `LuDialogService` under the hood when you're calling the `open` method.

Finally, a new `legacyOpen` method has been added to `LuModal` that behaves like the previous `open` method, if anything breaking happens for your use case.

-----
-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2586/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13675/ui-diff)

File: packages/ng/modal/modal.service.ts [hunk 1]
```diff
@@ -4,13 +4,37 @@ import { LuModalConfig } from './modal-config.model';
...
+			},
+			modal: !extendedConfig.noBackdrop,
+			alert: extendedConfig.undismissable,
+		});
+		/**
+		 * Submitaction, observable ou non,
+		 * 	Si observable => gérer loading/success/error
+		 * 	Sinon, rien à faire?
+		 *
+		 */
```
Comment (GuillaumeNury) [inline packages/ng/modal/modal.service.ts:33]:
```suggestion
```

**Review by GuillaumeNury — APPROVED** [review]
Nice!

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2586/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13678/ui-diff)

File: packages/ng/dialog/model/dialog-config.ts [hunk 2]
```diff
@@ -68,6 +71,10 @@ interface BaseLuDialogConfig<C> {
 	 * @param comp the instance of the component that's inside the dialog box.
 	 */
 	canClose?: (comp: C) => boolean | Observable<boolean>;
+
+	size?: 'XS' | 'S' | 'M' | 'L' | 'XL' | 'fitContent' | `maxContent` | 'fullScreen';
+
+	mode?: 'default' | 'drawer' | 'drawer-from-bottom';
```
Comment (XavCamp) [inline packages/ng/dialog/model/dialog-config.ts:77]: Pas d'option pour ne pas avoir le footer et ses boutons hardcodés ?

**Pull request merged**